source/operations/external-iam/configure-ad-ldap-external-identity-management.rst

```diff
@@ -99,40 +101,83 @@ An AD/LDAP user with no assigned policy *and* with membership in groups with no
 
    .. include:: /includes/k8s/steps-configure-ad-ldap-external-identity-management.rst
 
-
 .. Doing this the quick and dirty way. Need to revise later to be proper full includes via stepfiles
 
 .. cond:: linux or container or macos or windows
 
    .. _minio-external-identity-management-ad-ldap-configure:
 
+
    Procedure
    ---------
 
    1) Set the Active Directory / LDAP Configuration Settings
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   You can configure the AD/LDAP provider using either
-   environment variables *or* server runtime configuration settings. Both
-   methods require starting/restarting the MinIO deployment to apply changes. The
-   following tabs provide a quick reference of all required and optional
-   environment variables and configuration settings respectively:
+   Configure the AD/LDAP provider using one of the following:
+
+   * MinIO Client
+   * environment variables
+   * MinIO Console
+
+   All methods require starting/restarting the MinIO deployment to apply changes.
+
+   The following tabs provide a quick reference for the available configuration methods:
 
    .. tab-set::
 
+      .. tab-item:: MinIO Client
+
+         MinIO supports specifying the AD/LDAP provider settings using :mc:`mc idp ldap` commands.
+
+         For distributed deployments, the :mc:`mc idp ldap` command applies the configuration to all nodes in the deployment. 
+
+         The following example code sets *all* configuration settings related to configuring an AD/LDAP provider for external identity management.
+	 The minimum *required* settings are:
+
+         - :mc-conf:`server_addr <identity_ldap.server_addr>`
+         - :mc-conf:`lookup_bind_dn <identity_ldap.lookup_bind_dn>`
+         - :mc-conf:`lookup_bind_password <identity_ldap.lookup_bind_password>`
+         - :mc-conf:`user_dn_search_base_dn <identity_ldap.user_dn_search_base_dn>`
+         - :mc-conf:`user_dn_search_filter <identity_ldap.user_dn_search_filter>`
```

Comment:
This is more of a note:

So this will point the user ad the config settings instead of `mc idp ldap add <param>`. But we don't yet have dedicated pages for each param, so we don't have a choice here _yet_.

I think if we want to go properly into showcasing this command we're going to have to commit to breaking it all out into standalone pages.

Reply:
new issue for this https://github.com/minio/docs/issues/940